applications/CompressiblePotentialFlowApplication/python_scripts/apply_potential_flow_hessian_remeshing_process.py

```diff
@@ -64,7 +64,7 @@ def ExecuteFinalize(self):
 
     def __ComputeNodalVelocity(self):
 
-        nodal_velocity_process = CPFApp.ComputeNodalPotentialFlowVelocityProcess(self.main_model_part)
+        nodal_velocity_process = CPFApp.ComputeNodalValueProcess(self.main_model_part, ["VELOCITY"])
```

Comment:
@marcnunezc could you check that this PR does not break any of your examples where you use the potential_flow_hessian_remeshing_process? I couldn't find a test for this process, maybe we should add one in another PR before merging this.

Reply:
Ok i will check it!

Regarding the tests, it is true that we could add a test for the apply_hessian_remeshing_process.py. I did not add an specific test of it at first since it uses the metric_hessian_process.cpp and mmg_process.cpp, which are already covered in the MeshingApplication.